src/models.ts

```diff
@@ -1,3 +1,4 @@
+//Journeys models
```

Comment:
You could split this into two different files in a models directory so you don't need the comments to distinguish the different sections